currency.js

```diff
@@ -0,0 +1,61 @@
+var https = require('https')
+
+function convertCurrency(amount, fromCurrency, toCurrency, cb) {
+    var apiKey = '63a107cf0f91eaa9a44f'
+
+    fromCurrency = encodeURIComponent(fromCurrency)
+    toCurrency = encodeURIComponent(toCurrency)
+    var query = fromCurrency + '_' + toCurrency
+
+    var url =
+        'https://free.currconv.com/api/v7/convert?q=' +
+        query +
+        '&compact=ultra&apiKey=' +
+        apiKey
+
+    https
+        .get(url, function (res) {
+            var body = ''
```

Comment:
i would suggest changing the `var`s to `const`s where you can, it's considered best practice these days